### ISSUE TITLE
Running go-pro does not create a production config

### ISSUE THREAD

**Issue by Ganonmaster** [bunq/tinker_python#12]
Environment: Ubuntu 18.04, dependencies up-to-date.

1. Run setup with bash/curl pipe provided at bunq.com/api
2. Run ./go-pro
3. Scan the QR code and confirm in the app
4. go-pro returns with the title GO PRO title and the message "You're ready to use the PRODUCTION environment." and the help options with the --production flags.
5. No bunq-production.conf is created, so none of the commands work.

**Comment by Ganonmaster** [comment]
![image](https://user-images.githubusercontent.com/168445/43185905-85434e0e-8fed-11e8-9a1e-7a998a285cff.png)
No error, no production config.

Python 3.6.5, pip 18, pipenv 11.8.3.

**Comment by Ganonmaster** [comment]
Note that in the Bunq app, the API key still shows as "unassigned" as well.

**Comment by OGKevin** [comment]
Interesting, ill have a look and will let you know what my findings are.

**Comment by OGKevin** [comment]
hmm, i just ran it on macos and it seems to work as expected. 🤔 

Can you try to trace the execution of go pro and see if the bash script is actually running like it should. 

https://stackoverflow.com/questions/951336/how-to-debug-a-bash-script/

**Comment by Ganonmaster** [comment]
Found some debug output that might indicate where this is going wrong:

```
+ tinker/create_production_configuration.py --api-key <REMOVED>
/usr/bin/env: ‘.venv/bin/python -W ignore’: No such file or directory
```
It is making the assumption that my python binary is literally named `python`, when it actually is not.

![image](https://user-images.githubusercontent.com/168445/43191130-d458d128-8ffa-11e8-8916-190484c57704.png)

**Comment by OGKevin** [comment]
no, the bash script should create a dir ./.venv and in there there should indeed be a executable named `python` so it seems that the installation failed. 

it seems that 

https://github.com/bunq/tinker_python/blob/dfe9812d11a1b50d7aed715529e6ea5bd827ac9c/setup.sh#L146-L149

is not executed correctly then 🤔

**Comment by Ganonmaster** [comment]
![image](https://user-images.githubusercontent.com/168445/43191764-25f0ab40-8ffc-11e8-815e-536f8549c22f.png)

**Comment by Ganonmaster** [comment]
I think I found the problem. I had setup a custom alias for the default python version on ubuntu (because the default is 2.7)

**Comment by OGKevin** [comment]
by the look of the screen shot, are you in a virtual env ? might that be it ?

**Comment by Ganonmaster** [comment]
The virtualenv I enter is the one that the setup script creates. Is that not how it's supposed to work? Either way, I just tested without entering the virtualenv, and it makes no difference.

**Comment by OGKevin** [comment]
No, at least not for me on macos 🤔 

might it have something to do with https://github.com/bunq/tinker_python/issues/3 ?
